addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1335,15 +1361,27 @@ var SnippetsMenu = Widget.extend({
                     $els.attr('data-name', name).data('name', name);
                 }
 
+                // TODO Remove in master.
+                if ($snippet[0].querySelector('form')) {
+                    // In stable, make sure that existing FORM snippets cannot
+                    // be dropped in sanitized HTML model fields.
+                    $snippet.data('oeForbidSanitize', 'true');
+                }
+
+                const isSanitizeForbiddenClass = $snippet.data('oeForbidSanitize') ?
+                    " oe_forbid_sanitize" : "";
```

Comment:
Same here, remove that space IMHO :-)
It looks ugly when reading it, and we don't really care about it since it is part of a DOM only admins sees.